Options.py

```diff
@@ -844,7 +852,7 @@ class OptionList(Option[typing.List[typing.Any]], VerifyKeys):
     # If only unique entries are needed and input order of elements does not matter, OptionSet should be used instead.
     # Not a docstring so it doesn't get grabbed by the options system.
 
-    default: typing.Union[typing.List[typing.Any], typing.Tuple[typing.Any, ...]] = ()
+    default = ()
```

Comment:
Why not keep the more precises types? This could be `typing.ClassVar[typing.Iterable[str]]` for instance?

Reply:
Nvm, I should have read https://github.com/ArchipelagoMW/Archipelago/pull/2173 before commenting. All good 👍